doc/source/faq.rst

```diff
@@ -73,15 +73,15 @@ The name of the service has the following syntax: ``pmm-<type>-exporter-<port>``
 
 The location of the services depends on the service manager:
 
-+-----------------+---------------------+-----------------------------+
-| Service manager | Operating system    | Service location            |
-+=================+=====================+=============================+
-| ``systemd``     | CentOS 7 and RHEL 7 | :file:`/etc/systemd/system/`|
-+-----------------+---------------------+-----------------------------+
-| ``upstart``     | Debian and Ubuntu   | :file:`/etc/init/`          |
-+-----------------+---------------------+-----------------------------+
-| ``systemv``     | CentOS 6 and RHEL 6 | :file:`/etc/init.d/`        |
-+-----------------+---------------------+-----------------------------+
++-----------------+-----------------------------+
+| Service manager | Service location            |
++=================+=============================+
+| ``systemd``     | :file:`/etc/systemd/system/`|
++-----------------+-----------------------------+
+| ``upstart``     | :file:`/etc/init/`          |
++-----------------+-----------------------------+
+| ``systemv``     | :file:`/etc/init.d/`        |
++-----------------+-----------------------------+
 
 To see which service manager is used on your system, run ``sudo pmm-admin info``.
 
```

Comment:
Where is DSN stored?

For QAN, DSN is now stored in local qan-agent config files under /usr/local/percona/qan-agent. Not on QAN API.
No password are stored on the server by the way.